### ISSUE TITLE
Enjoying DuckDuckgo...

### ISSUE THREAD

**Issue by johannbg** [duckduckgo/Android#441]
Yes I shall share what I think, directly with you since users are asking me about this "feature" for some weird reason....

Any particular reason you have suddenly decide to no longer provide the duckduckgo android browser userbase the freedom of choice to share what they think about the browser with you and the ability to use it ( first time users ) before demanding end user feedback, that is that they share with you what they think before they can proceed with using the browser on atleast android devices?

Is this being done for some user tracking purposes since this is not an opt in "feature" but user must proceed with either 

"It needs work" or "I like it" 

before some users even get to try out the browser o_O 

What are you hoping to gain from this and 
is this not counter productive?

**Comment by CDRussell** [comment]
Hi @johannbg. 

>ability to use it ( first time users ) before demanding end user feedback

It's designed not to ask until you have been using the app for a bit; there's both a minimum numbers of searches criteria and a minimum number of days usage criteria. No one should be asked for feedback before they've had a chance to _use_ the app. If that's happening, let me know, as that is not as intended.

>demanding end user feedback, that is that they share with you what they think before they can proceed with using the browser

You'll probably be pleased to hear that we've already had the discussion internally and will likely be making the dialogs cancellable. As such, users who don't want to answer either way will be able to hit the back button and cancel out of it.

>this is not an opt in "feature"

As above, hopefully making it so that people can cancel it will make them happier. This isn't about tracking users. It's about ensuring we can improve the quality of the app. If we don't get anonymous feedback from users telling us what they don't like, we can't fix the pain points. Most users won't tell us if they stop liking the app; they'll just disappear silently. 

Also most users don't rate the app unless you explicitly ask them to. We know this from analysing the installation numbers and the ratings from the Play Store. But if do ask users to do rate the app, the number of reviews increase as does the overall app rating. This helps other people discover the app in the Play Store so that more people can benefit from the app's protection.

In short, it's a quick interruption that has a big impact, both in terms of us getting more feedback on areas for improvement and in terms of the app's rating.

**Comment by CDRussell** [comment]
_Closing this issue but happy to carry on the conversation._
